### ISSUE TITLE
disproportionately long limb

### ISSUE THREAD

**Issue by petkraw** [obophenotype/human-phenotype-ontology#3617]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
disproportionately long limb

### 2. Synonyms
long legs, long arms, marfanoid habitus

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
skeletal abnormality

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Marfan syndrome

**Comment by pnrobinson** [comment]
I think that this is a synonym for this:
https://hpo.jax.org/app/browse/term/HP:0001519
in the context of Marfan syndrome, the term Dolichostenomelia is used frequently -- this is a synonym of the term. 
I do not think that "disproportionately long limb" is a good synonym because it could mean that only one limb is affected. For this phenotype, one would choose a term such as this: https://hpo.jax.org/app/browse/term/HP:0010484
